index/miner/miner.go

```diff
@@ -256,13 +307,43 @@ func (mi *MinerIndex) Close() error {
 	return nil
 }
 
-func getPower(ctx context.Context, c API, addr string) (*PowerInfo, error) {
+func getPower(ctx context.Context, c API, addr string) (Power, error) {
 	mp, err := c.StateMinerPower(ctx, addr, nil)
 	if err != nil {
-		return nil, err
+		return Power{}, err
 	}
-	return &PowerInfo{
-		Power:      mp.MinerPower.Uint64(),
-		TotalPower: mp.TotalPower.Uint64(),
+	p := mp.MinerPower.Uint64()
+	tp := mp.TotalPower
+	return Power{
+		Power:    p,
+		Relative: float64(p) / float64(tp.Uint64()),
 	}, nil
 }
+
+func getStaticInfo(ctx context.Context, c API, h *fchost.FilecoinHost, lr iplocation.LocationResolver, addr string) (Static, error) {
+	si := Static{}
+	pid, err := c.StateMinerPeerID(ctx, addr, nil)
+	if err != nil {
+		return si, err
+	}
+	_, err = h.NewStream(ctx, pid, identify.ID)
+	if err != nil {
+		return si, err
+	}
+	si.Online = true
```

Comment:
If we could connect to the peer, is online.
This is used for the grafana dashboard, but can also be important as a reputation filter... since we could avoid others to fail connecting offline peers.
This may have to be tuned to scale... if the amount of miners is very big, we might relax the info as _last online time_ if we can't hold so many open connections. (just thinking out loud)